src/asgen/utils.d

```diff
@@ -370,6 +374,10 @@ body
             downloader.dataTimeout = dur!"seconds" (30);
             downloader.onReceive = (data) => onReceiveCb (dest, data);
             downloader.perform();
+            if ("last-modified" in downloader.responseHeaders) {
```

Comment:
If you want, you can optimize this slightly like this:
```D
immutable lastModifiedP = "last-modified" in downloader.responseHeaders;
if (lastModifiedP !is null) {
    auto lastmodified = *lastModifiedP;
    ret = parseRFC822DateTime(lastmodified);
}
```
=> one lookup instead of two.